assignment_2.java

```diff
@@ -0,0 +1,144 @@
+/***
+ *When the program is run, the user will first enter a string. After that, the program should
+ask the user which operation they want to execute on the string. The available operations are
+listed below: Append, CountWords, Replace, isPalindrome, Splice, Split,
+MaxRepeatingCharacter, Sort, Shift, and Reverse. Each method should perform its
+respective operation based on the user's input.
+Owner : Pratul_Agarwal
+Date of creation : 04/09/2024
+ */
+package assignment_java_programming;
+
+import java.util.Scanner;
+
+public class assignment_2 {
+
+    public static void main(String[] args) {
+    	constant constant = new constant();
+        Scanner scanner = new Scanner(System.in);
+        String tryagain = "y";
+        
+        while (tryagain.equalsIgnoreCase("y")) {
+        System.out.println(constant.ENTER_STRING6);

```

Comment:
No issue here, provided that ENTER_STRING6 is correctly defined in the constant class.